src/services/resource-service.ts

```diff
@@ -167,10 +167,21 @@ export const ResourceService = {
   getResourcesCategoriesNames: (): Promise<
     AxiosResponse<CategoryNameInterface[]>
   > => axiosClient.get(URLs.resources.resourcesCategories.getNames),
-  createResourceCategory: async (
-    params?: CreateCategoriesParams
-  ): Promise<AxiosResponse<Categories>> =>
-    await axiosClient.post(URLs.resources.resourcesCategories.post, params),
+  getResourcesCategoriesName: () => {
+    return baseService.request<CategoryNameInterface[]>({
+      method: 'GET',
+      url: URLs.resources.resourcesCategories.getNames
+    })
+  },
+  createResourceCategory: (
+    params: CreateCategoriesParams
+  ): Promise<Categories> => {
```

Comment:
It's already typed by baseService
```suggestion
  ) => {
```